### ISSUE TITLE
SyntaxError, line 150

### ISSUE THREAD

**Issue by Anarcomic** [jimtje/audm_scraper#2]
Hello and thanks for creating this script.

During installation, I get this error:
MacBooks-MBP:~ macbookpro$ python /Users/macbookpro/Downloads/audm_scraper-master/audm_scraper.py 
  File "/Users/macbookpro/Downloads/audm_scraper-master/audm_scraper.py", line 150
    concat_command = f"ffmpeg -nostats -loglevel 0 -y -f concat -safe 0 -i \"{tempfile}\" -c copy \"" \
                                                                                                                                                                       ^
SyntaxError: invalid syntax

Could you please help? I'm on Catalina latest, latest Python version too. What do you think I'm missing?

Kindest regards :)

**Comment by jimtje** [comment]
Is your ffmpeg installed and accessible in your PATH? You may want to brew install ffmpeg and see if the right executable is linked. 

And are you on Python 3.7 or 3.8? I haven't tested it above 3.8.

**Comment by Anarcomic** [comment]
The problem, I suspect, was related to the fact that I have two versions of Python installed, 2.7 and 3. I installed all the dependeices to 2.7, after which I still could get the error: 

`ImportError: No module named configparser`

In the end, I started the last step of the installation with Python3 + location of the script. It worked!

I have to admit that it's the first time I use python, homebrew, dependencies.... Took me some hours but it's so worth it. Send me your Paypal, maybe I buy you coffee/beer... :)
